R/data.table.R

```diff
@@ -1151,7 +1151,7 @@ replace_dot_alias <- function(e) {
               # Note that the NAMED(dt)>1 doesn't work because .Call
               # always sets to 2 (see R-ints), it seems. Work around
               # may be possible but not yet working. When the NAMED test works, we can drop allocwarn argument too
-              # because that's just passed in as FALSE from [<- where we know `*tmp*` isn't really NAMED=2.
+              # because that's just passed in as FALSE from [= where we know `*tmp*` isn't really NAMED=2.
```

Comment:
I see three places where you've changed [<- to [=, and two with $<- to $= (all in comments). I guess there could be other thing<- cases but didn't notice them.

Reply:
good eye, thanks. shouldn't be any in actual code (tests compile & run)

the key is if you use it in actual code, it's gotta be backticked or quoted (`[<-` or "[<-"), which is caught by the regex

Reply:
Anyway the regex could be fixed, previous character should be not-`[<[]`